reactioncommerce/reaction
# Fix Exception in delivering result of invoking 'email/verifySettings'

This resolves #2464

### Steps to test
1. Sign in as admin
1. Open email settings on the dashboard and update settings
1.  Close the dashboard panel
1.  Observe that there is no error on the console
